Dockerfile

```diff
@@ -8,23 +8,89 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-ARG CUDA_VERSION=9.2
-ARG LINUX_VERSION=ubuntu16.04
-ARG CC_VERSION=5
-ARG CXX_VERSION=5
+ARG FROM_IMAGE=gpuci/miniconda-cuda
+ARG CUDA_VER=10.1 
+ARG LINUX_VERSION=ubuntu18.04
+ARG CC_VERSION=7
+ARG IMAGE_TYPE=devel
 ARG PYTHON_VERSION=3.6
 
-FROM gpuci/rapidsai-base:cuda${CUDA_VERSION}-${LINUX_VERSION}-gcc${CC_VERSION}-py${PYTHON_VERSION}
+FROM ${FROM_IMAGE}:${CUDA_VER}-${IMAGE_TYPE}-${LINUX_VERSION}
 
-# Install htslib dependencies
-RUN apt-get update
-RUN apt-get install -y tabix \
-        zlib1g-dev \
-        libbz2-dev \
-        liblzma-dev \
-        libcurl4-gnutls-dev \
-        wget \
-        libssl-dev      # VariantWorks `cyvcf2` dependency
+# Capture argument used for FROM
+ARG CC_VERSION
```

Comment:
I see `ARG CC_VERSION` and `ARG PYTHON_VERSION` near line 14 as well. Can those be reused here?

Reply:
You're right, no need to declare these at all before the `FROM` command.